src/graphql/v2/dataSource.js

```diff
@@ -413,6 +407,22 @@ class FhirDataSource extends DataSource {
         }
         return combinedMeta;
     }
+
+    /**
+     * Concatenate code and/or display
+     * @param {Coding | undefined} foundCombinedMetaTag
+     * @param {Coding | undefined} metaTag
+     * @returns {Object}
+     */
+    updateCombinedMetaTag(foundCombinedMetaTag, metaTag) {
```

Comment:
THe function does not add any value to the class. It more of a util required by getBundleMeta() function. Consider moving it inside getBundleMeta().